shipstation_integration/shipments.py

```diff
@@ -158,3 +184,55 @@ def create_delivery_note(shipment, sales_invoice):
 	dn.submit()
 	frappe.db.commit()
 	return dn
+
+
+def create_shipment(
+	shipment: "ShipStationOrder",
+	delivery_note: "DeliveryNote",
+	store: "ShipstationStore",
+):
+	shipment_doc: "Shipment" = make_shipment(delivery_note.name)
+	shipment_doc.update(
+		{
+			# todo: change description of content to shipment items.
+			"shipment_id": shipment.shipment_id,
+			"pickup_date": shipment.create_date,
+			"carrier": shipment.carrier_code,
+			"carrier_service": shipment.service_code,
+			"awb_number": shipment.tracking_number,
+			"service_provider": "Shipstation",
+			"incoterm": "DAP (Delivered At Place)",
+			"shipstation_store_name": store.store_name,
+			"shipstation_order_id": shipment.order_id,
+			"marketplace": store.marketplace_name,
+			"marketplace_order_id": shipment.order_number,
+		}
+	)
+
+
+	if shipment.shipment_items:
+		shipment_items = [shipment_item.get("name") for shipment_item in shipment.get("shipment_items")]
+
+		shipment_doc.update({
+			"desciption_of_content": ', '.join(shipment_items)
```

Comment:
See if we can add the quantity in as well. That'll be good info to have here.

Reply:
Okay